File: app/src/main/java/org/eva/jason140520/view/LaunchActivity.java
package org.eva.jason140520.view;/**
 * Copyright (C), 2020, Jason
 * Author: JiangLin
 * Date: 2020/1/2 23:00
 * History:
 * <author> <time> <version> <desc>
 * 作者姓名 修改时间 版本号 描述
 */

import android.os.Bundle;

import androidx.annotation.Nullable;
import androidx.appcompat.app.AppCompatActivity;

import org.eva.jason140520.R;

/**
 * @ClassName: LaunchActivity
 * @Description: app启动页Activity
 * @Author: JiangLin
 * @Date: 2020/1/2 23:00
 */
public class LaunchActivity extends AppCompatActivity {

    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_launch);
    }
}

File: app/src/main/java/org/eva/jason140520/view/LaunchActivity.kt
package org.eva.jason140520.view

import android.os.Bundle
import android.view.Window
import androidx.appcompat.app.ActionBar
import androidx.appcompat.app.AppCompatActivity
import org.eva.jason140520.R

/**
 * @ClassName:      MainActivity
 * @Description:     java类作用描述
 * @Author:         JiangLin
 * @CreateDate:     2019/12/30 23:39
 * @UpdateUser:     JiangLin
 * @UpdateDate:     2019/12/30 23:39
 * @UpdateRemark:   新建类
 * @Version:        1.0
 */
class LaunchActivity: AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        //supportActionBar?.hide()//隐藏ActionBar
        setContentView(R.layout.activity_launch)
    }
}